app/routes/cdo/search/basic.js

```diff
@@ -0,0 +1,45 @@
+const { routes, views } = require('../../../constants/search')
+const { getSearchCriteria, setSearchCriteria, getSearchResults, setSearchResults } = require('../../../session/cdo/search')
+const ViewModel = require('../../../models/cdo/search/basic')
+const searchSchema = require('../../../schema/portal/search/basic')
+const { doSearch } = require('../../../api/ddi-index-api/search')
+const { admin } = require('../../../auth/permissions')
+
+module.exports = [{
+  method: 'GET',
+  path: routes.searchBasic.get,
+  options: {
+    auth: { scope: [admin] },
+    handler: async (request, h) => {
+      const searchCriteria = getSearchCriteria(request)
+      const searchResults = getSearchResults(request)
+      return h.view(views.searchBasic, new ViewModel(searchCriteria, searchResults))
+    }
+  }
+},
+{
+  method: 'POST',
+  path: routes.searchBasic.post,
+  options: {
+    auth: { scope: [admin] },
+    validate: {
+      payload: searchSchema,
+      failAction: async (request, h, error) => {
+        const searchCriteria = { ...getSearchCriteria(request), ...request.payload }
+        const searchResults = getSearchResults(request)
+        return h.view(views.searchBasic, new ViewModel(searchCriteria, searchResults, error)).code(400).takeover()
+      }
+    },
+    handler: async (request, h) => {
```

Comment:
I figured someone might want to go back to the search results page after viewing (say) a dog, rather than having to navigate to 'search' then enter identical criteria, to see the same results as before - hence persisting.

Reply:
Yes I understand now. I think implementing the change to GET with criteria in query string should allow us to persist without use of session.

I don't think it's complicated enough to justify making it a POST.